CrimeMonitor/CrimeMonitor.cs

```diff
@@ -654,6 +660,60 @@ public void _RemoveRecord(FactionRecord record)
             }
         }
 
+        private void AddCrimeToRecord(FactionRecord record, FactionReport report)
+        {
+            long total = record.fines + record.bounties + report.amount;
+            FactionRecord powerRecord = GetRecordWithFaction(record.allegiance);
+
+            if (powerRecord != null || total > 2000000)
```

Comment:
Comment on the 2000000? As you noted today, that wasn't obvious and was something you had to discover.

Reply:
Done.